README.md

```diff
@@ -60,6 +62,30 @@ kubectl crossplane stack list
 kubectl crossplane stack uninstall 'myname-mysubname'
 ```
 
+### Trace command
+
+Trace command aims to ease debugging and troubleshooting process by providing a holistic view for a particular object.
+It finds the relevant objects for a given one and provides detailed information.
+
+```
+$ kubectl crossplane trace TYPE[.GROUP] NAME [-n| --namespace NAMESPACE]
```

Comment:
Recently we've moved away from putting the `$` shell prompt at the beginning of the line so that it's easier to copy and paste blindly into a terminal session. Would you mind removing the leading `$` characters from the code examples?